plugins/modules/nxos_user.py

```diff
@@ -407,7 +392,14 @@ def main():
 
     if commands:
         if not module.check_mode:
-            load_config(module, commands)
+            responses = load_config(module, commands)
+            for resp in responses:
+                if resp.startswith("WARNING: "):
+                    result["warnings"].extend(
+                        [x.replace("WARNING: ", "") for x in resp.splitlines()]
+                    )
```

Comment:
I don't know if warnings can come after other response lines, but this seems more robust in general.

```suggestion
                result["warnings"].extend([
                    x[9:] for x in resp.splitlines()
                    if x.startswith("WARNING: ")
                ])
```

Reply:
I hadn't thought of this. This would probably not happen, but in any case what you suggested looks much better. I'll push these changes.